content/en/security/application_security/enabling/compatibility/nodejs.md

```diff
@@ -6,46 +6,164 @@ type: multi-code-lang
+|                     | Linux (musl)          | arm64, x64 | Alpine 3.10                              |
+|                     | macOS                 | arm64, x64            | Yosemite (10.10)                         |
+|                     | Windows               |  x64             | Windows 8.1, Windows Server 2012         |
+
+
+
+
+
+### Web framework compatibility
+
+- Attacker source HTTP request details
+- Tags for the HTTP request (status code, method, etc)
+- Distributed Tracing to see attack flows through your applications
+
+##### ASM Capability Notes
+- Open Source Vulnerability Detections is supported on all frameworks
+
+
+| Framework                  | Versions   | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
+| ----------------------- | ---------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
+| express | >=4| {{< X >}}  |{{< X >}}  | {{< X >}} |
+
 
-| ASM capability                   | Minimum NodeJS tracer version |
-| -------------------------------- | ----------------------------|
-| Threat Detection <br/> --> Business logic API  | 3.13.1 <br/> |
-| Threat Protection <br/> --> IP blocking <br/> --> Suspicious request blocking <br> --> User blocking | <br/> --> 3.11.0 <br/> --> 3.19.0 <br/> --> 3.11.0 |
-| Vulnerability Management <br/> --> Open source vulnerability detection <br/> | <br/> 2.23.0 for NodeJS 12+, or 3.10.0 for NodeJS 14+ |
-| Vulnerability Management <br/> --> Custom code vulnerability detection <br/> | <br/> 2.32.0 for NodeJS 12+, or 3.19.1 for NodeJS 14+ |
 
-The minimum tracer version to get all supported ASM capabilities for Node.js is 3.19.1.
 
-**Note**: Threat Protection requires enabling [Remote Configuration][2], which is included in the listed minimum tracer version.  
 
 <div class="alert alert-info">If you would like to see support added for any of the unsupported capabilities or for your Node.js framework, let us know! Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send details</a>.</div>
 
 
+### Networking framework compatibility
+
+
+**Networking tracing provides:**
+
+- Distributed tracing through your applications
+- Request-based blocking
+
+##### ASM Capability Notes
+- Open Source Vulnerability Detection is supported on all frameworks
+
+
+
+| Framework                | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
+| ------------------------ | ----------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
+| dns     | {{< X >}} |  |  |
+| fs      |  {{< X >}} |  |  {{< X >}}  |
+| http    |  {{< X >}} |  {{< X >}}  |  {{< X >}}  |
+| https   |  {{< X >}} |  {{< X >}} |  |
+| http2   |  {{< X >}} |  {{< X >}} |  |
```

Comment:
fs is file system, nothing related with network.
We are doing something to http or https, but nothing for `dns` or `http2`

```suggestion
| Framework                | Threat Detection supported? | Threat Protection supported? | Custom Code Vulnerability Detection supported? |
| ------------------------ | ----------- | --------------- | ---------------------------------------------- | ---------------------------------------------- |
| http    |  {{< X >}} |  {{< X >}}  |  {{< X >}}  |
| https   |  {{< X >}} |  {{< X >}} |  |
```

Reply:
updated